sdks/python/src/opik/integrations/llama_index/callback.py

```diff
@@ -98,6 +113,18 @@ def on_event_start(
         # Compute the span input based on the event payload
         span_input = event_parsing_utils.get_span_input_from_events(event_type, payload)
 
+        if self._opik_trace_data.project_name != self._project_name:
+            if self._project_name is not None:
+                LOGGER.warning(
+                    "You are attempting to log data into a nested span under "
```

Comment:
Looks like we are using this message in multiple places, let's extract this string to a constant in `logging_messages.py`